app/views/browse/stateful-set.html

```diff
@@ -0,0 +1,178 @@
+                ng-if="statefulSet"
+                ng-show="resourceGroupVersion.resource | canIDoAny">
+                <button
+                  type="button"
+                  class="dropdown-toggle btn btn-default actions-dropdown-btn hidden-xs"
+                  data-toggle="dropdown">
+                  Actions
+                  <span class="caret" aria-hidden="true"></span>
+                </button>
+                <a
+                  href=""
+                  class="dropdown-toggle actions-dropdown-kebab visible-xs-inline"
+                  data-toggle="dropdown">
+                  <i class="fa fa-ellipsis-v"></i><span class="sr-only">Actions</span>
+                </a>
+                <ul class="dropdown-menu actions action-button">
+                  <!-- TODO: kinda works, but cant seem to edit. neat. -->
+                  <li ng-if="resourceGroupVersion | canI : 'update'">
+                    <a ng-href="{{statefulSet | editYamlURL}}" role="button">Edit YAML</a>
+                  </li>
+                  <li ng-if="resourceGroupVersion | canI : 'delete'">
+                    <!-- TODO: Warn about replicas? deleting statefulSets doesn't delete the pods -->
+                    <!-- TODO: hpa-list attrib? figure out autoscaling -->
+                    <delete-link
+                      kind="statefulSet"
+                      group="apps"
+                      resource-name="{{statefulSet.metadata.name}}"
+                      project-name="{{statefulSet.metadata.namespace}}"
+                      replicas="statefulSet.status.replicas"
+                      alerts="alerts">
+                    </delete-link>
+                  </li>
+                </ul>
+              </div>
+            </h1>
+            <labels
+              labels="statefulSet.metadata.labels"
+              clickable="true"
+              kind="statefulSets"
+              project-name="{{statefulSet.metadata.namespace}}"
+              limit="3"></labels>
+          </div>
+        </div>
+      </div><!-- /middle-header-->
+
+      <div class="middle-content" persist-tab-state>
+        <div class="container-fluid">
+         <div ng-if="!loaded">Loading...</div>
+          <div class="row" ng-if="loaded">
+            <div class="col-md-12">
+              <uib-tabset>
+                <uib-tab active="selectedTab.details">
+                  <uib-tab-heading>Details</uib-tab-heading>
+
+                  <div class="resource-details">
+                    <dl class="dl-horizontal left">
+                      <dt>Status:</dt>
+                      <dd>
+                        <!-- the deploymentStatus filter seems to work fine here -->
+                        <status-icon status="statefulSet | deploymentStatus"></status-icon>
+                        {{statefulSet | deploymentStatus}}
+                      </dd>
+                      <dt ng-if-start="statefulSet | annotation:'deploymentStatusReason'">Status Reason:</dt>
```

Comment:
i dont think this annotation exists for stateful sets, this was specific to RCs created by deployment configs